Misc/Positions_Area Positions 4 External.txt

```diff
@@ -72,9 +67,6 @@ LFRR_W_CTR:Brest Control:127.860:BRW:C:LFRR:CTR:LFRR:CTR:5640:5657
 
 LFFM_CTR:France Military Control:128.375:FM:C:LFFM:CTR:FRANCE:MIL::
 
-;France - Withdrawn 2023/11
-LFFF_CTR:Paris Control:128.100:P:C:LFFF:CTR:LFFF:CTR:5700:5767
-
 LFEE_N_CTR:Reims Control:127.550:EEN:C:LFEE:CTR:LFEE:NORTH:5700:5767
```

Comment:
All lines from `;France - Withdrawn 2023/11` to `;Shannon` are part of the withdrawn section and can be removed.